modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/display/context/AnalyticsReportsDisplayContext.java

```diff
@@ -131,129 +85,12 @@ public String getLiferayAnalyticsURL() {
 
 	private Map<String, Object> _getContext() {
 		return HashMapBuilder.<String, Object>put(
-			"defaultTimeRange",
-			() -> {
-				TimeSpan defaultTimeSpan = TimeSpan.of(
-					TimeSpan.defaultTimeSpanKey());
-
-				TimeRange defaultTimeRange = defaultTimeSpan.toTimeRange(0);
-
-				return HashMapBuilder.<String, Object>put(
-					"endDate",
-					DateTimeFormatter.ISO_DATE.format(
-						defaultTimeRange.getEndLocalDate())
-				).put(
-					"startDate",
-					DateTimeFormatter.ISO_DATE.format(
-						defaultTimeRange.getStartLocalDate())
-				).build();
-			}
-		).put(
-			"defaultTimeSpanKey", TimeSpan.defaultTimeSpanKey()
-		).put(
 			"endpoints",
 			HashMapBuilder.<String, Object>put(
 				"getAnalyticsReportsData",
```

Comment:
From @carloslancha feedback:
>To me anything starting with get is a method (a getter), getAnalayticsReportsDataURL sounds like something that will return you an URL. Something like analyticsReportsDataURL. analyticsReportsDataFetchURL, analyticsReportsDataEndpointURL, makes more sense to me in this case.

Maybe we should rename this (also in the frontend side) to something like he proposed 🤔  `analyticsReportsDataURL`?

Reply:
I totally agree, could you open an impeding with this?

Reply:
Yes! Do you want one impedibug for both changes or split into two?

Reply:
One impedibug looks ok for me :)

Reply:
https://issues.liferay.com/browse/LPS-119831